### ISSUE TITLE
Bounding boxes applied to challenge through API are not applied to imported tasks.

### ISSUE THREAD

**Issue by atiannicelli** [maproulette/maproulette2#908]
I use the API to create a challenge with a set of high priority bounding boxes to make issues found in those boxes high priority. I then upload tasks to the challenge through the API as well. When I do this it seems that the priority is not applied to the imported tasks when uploading. If I then alter (or maybe just save) the challenge from the GUI the high priority bounding boxes are then applied.

I believe that the priority rules should be applied to tasks as they are uploaded and they apparently are not currently being applied.

Here is a movie I made to demonstrate the problem symptom
https://www.youtube.com/watch?v=6S5ZRl6p9zs

In other words - if I do the following with the API:
1) create a challenge with a high priority bounding box.
2) add tasks to the challenge that are inside that bounding box
The added tasks do not get the high priority bounding box applied as they should.

**Comment by atiannicelli** [comment]
I just confirmed that if I load tasks and see that the priority rules are not being applied (at this point I see ALL my tasks are low priority - `2`) and then if I issue a `PUT /challenge/{id}` command with NO changes to the body of the challenge JSON then nothing happens (All tasks are still low priority even though some tasks land within bounding boxes defined as high priority).
If, however, I change the `defaultPriority` from 2 to 1 THEN I see most tasks change from Low to Medium priority and the ones that are within the high priority bounding boxes change from Low to High.

So it seems the only way to work around this issue at the moment is to upload all my tasks and then change something about the challenge settings.
